### PR TITLE
metrics: Add remote networking latency test

### PR DESCRIPTION
**Pull request by GabyCT**
This will measure latency across containers in two different hosts.

Fixes ##732

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by GabyCT** [comment]
@iphutch could you please review this? thanks

File: metrics/network/remote_network/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Clear Containers remote networking test
 
-- The`remote-networking-iperf3.sh` script measures bandwidth, jitter or parallel bandwidth using `iperf3`
-using a remote setup. Host A will run a container that will act as a server while host B will run a
+- The`remote-networking-iperf3.sh` script measures bandwidth, jitte, latency or parallel bandwidth using
```
Comment (iphutch) [inline metrics/network/remote_network/README.md]: Typo: "jitte"

**Comment by GabyCT** [comment]
@iphutch thanks, changes applied

File: metrics/network/remote_network/remote-networking-iperf3.sh [hunk 2]
```diff
@@ -129,11 +147,16 @@ function main {
 			parallel_test="1"
 			remote_network_parallel_iperf3
 		;;
+		l)
+			latency_test="1"
```
Comment (MarioCarrilloA) [inline metrics/network/remote_network/remote-networking-iperf3.sh]: who will consume these vars (`var="1"`)?

Comment (GabyCT) [reply metrics/network/remote_network/remote-networking-iperf3.sh]: remote_network_latency

Comment (GabyCT) [reply metrics/network/remote_network/remote-networking-iperf3.sh]: remote_network_latency

**Comment by devimc** [comment]
lgtm

**Pull request merged**